db/migrate/20161109150714_add_templates_to_resolve_url.rb

```diff
@@ -0,0 +1,21 @@
+class AddTemplatesToResolveUrl < ActiveRecord::Migration[5.0]
+  def up
+    Agents::WebsiteAgent.find_each do |agent|
+      keys = agent.event_keys
+      next if keys.nil? || !keys.include?('url') || agent.options.key?('template')
+      agent.options['template'] = keys.each_with_object({}) { |key, template|
+        case key
+        when 'url'
+          template[key] = '{{ url | to_uri: _response_.url }}'
+        else
+          template[key] = "{{ #{key} }}"
```

Comment:
Can't we skip this because the key will be provided in each outgoing event anyway?